README.md

```diff
@@ -37,22 +37,19 @@
 
   * Compile the Go code and create the Cluster Operator images (both Go and Ansible):
     * `make images`
-  * Idempotently deploy cluster operator to the OpenShift cluster.
-    * `ansible-playbook contrib/ansible/deploy-playbook.yaml`
-  * If your image changed, but the kubernetes config did not (which is usually the case), you should pods appropriately:
+  * Deploy cluster operator to the OpenShift cluster you're currently logged into:
```

Comment:
You may want to mention that you need to be a cluster admin to do this, either by using 'oc login -u system:admin' or 'oc login -u admin' (where you previously granted admin the cluster-admin role)